tools/server-spring/src/main/java/org/eclipse/rdf4j/http/server/repository/statements/StatementsController.java

```diff
@@ -393,7 +393,7 @@ private ModelAndView getAddDataResult(Repository repository, HttpServletRequest
 			if (replaceCurrent) {
 				repositoryCon.clear(contexts);
 			}
-			repositoryCon.add(in, baseURI.toString(), rdfFormat, contexts);
+			repositoryCon.add(in, baseURIString, rdfFormat, contexts);
```

Comment:
I was wondering what this does  when `baseURIString` is `null`. But according to the javadoc at least, the method should accept `null` as a value here, so all good. I assume we originally put this dummy value there because it _didn't_ accept `null` at the time, but then we changed that behavior at some point and forgot to take this out.